frontend/src/components/OdhDocListItem.tsx

```diff
@@ -93,7 +93,7 @@ const OdhDocListItem: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFav
       </div>
       <div className="odh-list-item__doc-text">
         {odhDoc.spec.appDisplayName ? (
-          <div id={odhDoc.spec.appDisplayName} className="odh-list-item__doc-title">
```

Comment:
should this be removed or just changed to be unique?

Reply:
Updated to use a unique ID here as discussed in scrum today @Gkrumbach07!